hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -451,7 +451,9 @@ public AbfsConfiguration(final Configuration rawConfig, String accountName)
   }
 
   public Trilean getIsNamespaceEnabledAccount() {
-    return Trilean.getTrilean(isNamespaceEnabledAccount);
+    String isNamespaceEnabledAccountString
```

Comment:
Hi, it seems fine to keep this name as well as it clarifies that it is the Account's property that we are talking about not a Filesystem's property.

Besides, this has been the day 0 code and changing it would lead to a lot of code changes everywhere where getter and setters are used. We have followed same naming structure for this throughout driver code.

Would love to know your thoughts here.

Reply:
"I am just asking you to rename the newly introduced local variable **isNamespaceEnabledAccountString** to **isNamespaceEnabled**, not the method name.

Reply:
Ohh okay...
Will do that.
Thanks for clarifying.

Reply:
Going with @bhattmanish98 's suggestion below to get rid of local variable itself.